fe/fe-core/src/main/java/org/apache/doris/qe/ConnectContext.java

```diff
@@ -263,6 +263,9 @@ public ConnectContext(StreamConnection connection) {
             mysqlChannel = new DummyMysqlChannel();
         }
         sessionVariable = VariableMgr.newSessionVariable();
+        if (isMajorVersionUpgrade) {
```

Comment:
we need global set, here just change the connect way, if fe restart will change the logic